hack/local-up-cluster.sh

```diff
@@ -436,10 +437,8 @@ function start_apiserver {
 
     # Admission Controllers to invoke prior to persisting objects in cluster
     #
-    # ResourceQuota must come last, or a creation is recorded, but the pod may be forbidden.
-    ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount${security_admission},DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,PodPreset
-    # This is the default dir and filename where the apiserver will generate a self-signed cert
-    # which should be able to be used as the CA to verify itself
+    # The order defined here dose no matter.
+    ENABLE_ADMISSION_PLUGINS=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount${security_admission},DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,PodPreset
```

Comment:
aren't these enbabled by default?

Reply:
No, currently only `NamespaceLifecycle` default enabled. I am not sure which plugins should be enabled by default.

Reply:
If the cluster starts up with the default set, let's keep everything on IMO. Gives more realistic testing.

Reply:
What I change here is all replace `admission-control` with `enable-admission-plugins`. This line also applies.

Reply:
Yes, but `admission-control` needed a lot of plugin. `enable-admission-plugins` only need additional plugins. So the list should be much shorter, shouldn't it?

Reply:
Yes, the default enabled `NamespaceLifecycle` can be removed.